src/ssl.c

```diff
@@ -15996,7 +16018,49 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
             ctx->CBClientCert = cb;
         }
     }
-    #endif /* OPENSSL_ALL || OPENSSL_EXTRA || WOLFSSL_NGINX || WOLFSSL_HAPROXY */
+
+    void wolfSSL_CTX_set_cert_cb(WOLFSSL_CTX* ctx,
+        CertSetupCallback cb, void *arg)
+    {
+        WOLFSSL_ENTER("wolfSSL_CTX_set_cert_cb");
+        if (ctx == NULL)
+            return;
+
+        ctx->certSetupCb = cb;
+        ctx->certSetupCbArg = arg;
+    }
+
+    /**
+     * Internal wrapper for calling certSetupCb
+     * @param ssl
+     * @return 0 on success
+     */
+    int CertSetupCbWrapper(WOLFSSL* ssl)
+    {
+        int ret = 0;
+        if (ssl->ctx->certSetupCb != NULL) {
+            WOLFSSL_ENTER("CertSetupCbWrapper");
```

Comment:
Should ENTER be moved above the callback NULL check or maybe just removed/combined with MSG?

Reply:
Removing `WOLFSSL_ENTER`. Leaving just the `WOLFSSL_MSG`.